tests/integration/test_engine_integration.py

```diff
@@ -150,6 +147,27 @@ def test_measured_parameter(self, setup_eng):
 
         eng.run(prog)
 
+    @pytest.mark.backends("tf")
+    @pytest.mark.skipif("BATCHED" not in os.environ, reason="Test for when combining batched samples")
```

Comment:
How come we went with os.environ over previous method?

Reply:
The previous way (using the `@property` decorated function) didn't seem to work. It was outside a class, where I believe `@property` isn't supported. When I tried using it, it didn't work as expected, so I think it was broken before. I changed it to check for the environment variable `BATCHED` instead since I couldn't figure out another nicer way (except for perhaps just having an `if batch_size is not None` inside the test.

Reply:
I seem to remember it used to be that the testing framework itself picked these up from the environment and made them accessible via fixtures (long time ago, so things might have changed since then)